compiler/cpp/src/thriftl.ll

```diff
@@ -155,7 +155,8 @@ literal_begin (['\"])
 "include"            { return tok_include;              }
 "void"               { return tok_void;                 }
 "bool"               { return tok_bool;                 }
-"byte"               { return tok_byte;                 }
+"byte"               { return tok_i8;                   }
```

Comment:
could we somehow generate warning here? "using byte type is deprecated as it might lead to misunderstanding in signess" or something